client/src/entrypoints/admin/draftail.js

```diff
@@ -64,3 +64,10 @@ if (!window.Draftail || !window.draftail) {
     window.draftail.initEditor(`#${id}`, detail, document.currentScript);
   });
 }
+
+if (window.telepath) {
+  window.telepath.register(
+    'wagtail.widgets.DraftailRichTextArea',
+    draftail.DraftailRichTextArea,
+  );
+}
```

Comment:
Here and in `date-time-chooser.js`, are there cases where this module is loaded before `window.telepath` is available? Does that mean this will just fail silently (the widget doesn't get registered), and is that what we want?

Reply:
There shouldn't be any cases of that on admin templates that use admin_base.html, but I think it's worth accounting for the possibility that people might import draftail.js and date-time-chooser.js on their own non-admin pages - unlike Telepath itself, I don't think we'd discourage that. In that case, they'd get the functionality of Draftail / the date pickers without the Telepath integration.